File: problemas/iniciante/2791/2791.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);

        String[] numeros = in.readLine().trim().split(" ");
        
        for(int i = 0; i < 4; ++i){
            if(Integer.parseInt(numeros[i]) == 1){
                System.out.println(i + 1);
                break;
            }
        }
    }
}

File: problemas/iniciante/2791/2791.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = '';
let lineCount = 0;

process.stdin.on('data', (chunk) => {
  input += chunk;
});

process.stdin.on('end', () => {
  var lines = input.split('\n');

  let C = lines.shift()!.trim().split(' ').map(x => parseInt(x));

  for (let i = 0; i < 4; ++i) {
    if (C[i] == 1) {
      console.log(i + 1);
      break;
    }
  }
});